terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -305,32 +301,28 @@ locals {
         "location": "gds://production/reference-data/dragen_hash_tables/v8/hg38/altaware-cnv-anchored/hg38-v8-altaware-cnv-anchored.tar.gz"
       },
       "qc_reference_samples":[
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00061_MDX190049_L1900440/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00061"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00188_MDX190169_L1900877/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00188"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00199_MDX190193_L1900920/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00199"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00236_MDX190219_L1901023/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00236"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00238_MDX190231_L1901028/analysis/dragen_wts/2021-11-29--3.9.3/SBJ00238"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00028_PRJ190190_L1900423/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00028"
-          }
-      ],
-      "cl_config": "{\"sample_names_replace\":{\"L1900440\":\"Ref_1\",\"L1900877\":\"Ref_2\",\"L1900920\":\"Ref_3\",\"L1901023\":\"Ref_4\",\"L1901028\":\"Ref_5\",\"L1900423\":\"Ref_6\"}}",
+            {
+                "class": "Directory",
+                "location": "gds://production/reference-data/multiqc/2022-08-17--3.9.3/SBJ01563"
```

Comment:
Yep, ditto mid-path `/dragen_wts/`

Reply:
updated in [bc39467](https://github.com/umccr/infrastructure/pull/243/commits/bc39467cb32d623fc7548cbb8354c283007edd07)